Firmware/ultralcd.cpp

```diff
@@ -1022,7 +1022,7 @@ void lcd_pause_print()
 
     SERIAL_ECHOLNRPGM(MSG_OCTOPRINT_PAUSED);
 
-    print_job_timer.pause();
+    print_job_timer.pause(true);
```

Comment:
I don’t think this is the correct approach to fixing the issue. 

This change doesnt fix numerous issues with conditional logic in the firmware when the user doesnt start the timer to begin with.

Reply:
I haven't looked into it yet, but one idea I have is to expand this to cover paused host prints:

```cpp
enum class PrinterState : uint8_t
{
    NotReady = 0,                 //Lowest state to simplify queries
    IsReady = 1,                  //
    Idle = 2,
    SDPrintingFinished = 3,
    HostPrintingFinished = 4,
    IsSDPrinting = 5,
    IsHostPrinting = 6,
};
```

For example adding `isPrintingPaused = 7`, to `lcd_pause_print()`. That should cover M601. And we should change also various other conditional logic to use `isPrintingPaused` instead of `print_job_timer.isPaused()`. Especially for Host prints where the print job timer is optional.

Reply:
Yes, this will not fix conditional logic in the firmware when the user doesn't start the timer to begin with. This change only restores the exact behaviour as it was before the new functionality was added. I think fixing the conditional logic could be a separate task.

Reply:
@sarusani I looked a bit into how Marlin 2.1 is handling this. They have `did_pause_print`

```cpp
/**
 * Printing is paused according to SD or host indicators
 */
bool printingIsPaused() {
  return did_pause_print || print_job_timer.isPaused() || IS_SD_PAUSED();
}
```

It's set when the `//action:paused` is sent to Octoprint. I will try see if I can get this working this weekend :)